util/pace/util/_legacy_restart.py

```diff
@@ -1,7 +1,11 @@
-import copy
 import os
 from typing import BinaryIO, Generator, Iterable
 
+from pace.util.initialization import (
```

Comment:
I'm concerned about any changes to this file. Are you able to write this so that this file is exactly the same as on main, but instead you have code in `pace.driver.initialization` that e.g. assigns variables to the correct driver state variable from the right fortran name?

Reply:
I think this is resolved? maybe?